source/usage-examples/deleteOne.txt

```diff
@@ -45,5 +45,30 @@ to match only movies with a title of type ``string``.
 
 .. include:: /includes/connect-guide-note.rst
 
-.. literalinclude:: /code-snippets/usage-examples/deleteOne.js
-  :language: javascript
+.. tabs::
+
+  .. tab:: JavaScript
+     :tabid: javascript
+
+     .. literalinclude:: /code-snippets/usage-examples/deleteOne.js
+        :language: javascript
+        :linenos:
+
+  .. tab:: TypeScript
+     :tabid: typescript
+
+     .. literalinclude:: /code-snippets/usage-examples/deleteOne.js
+        :language: javascript
+        :linenos:
+
+.. note:: Identical Code Snippets
+
+   The JavaScript and TypeScript code snippets above are identical. There is no
+   added value from using TypeScript in this use case.
```

Comment:
I would suggest that instead of saying "There is no added value from using TypeScript" we say "There are no specific TypeScript features relevant to this use case", or something along those lines. (TypeScript will still provide type hinting in the editor, just nothing that the developers need to actively think about in this example)

Reply:
Good rephrase! How is "There are no TypeScript specific features of the driver relevant to this use case"?

Reply:
Sure, sounds good to me :)